src/node/glossary.ts

```diff
@@ -0,0 +1,12 @@
+import { SharedOptions } from '../sharedOptions'
+import { RequestHandlersList } from '../setupWorker/glossary'
+
+export type ListenOptions = SharedOptions
+
+export interface ServerInstance {
```

Comment:
Let's name this interface `SetupServerApi`, to be aligned with how a similar `setupWorker` return type is called (`SetupWorkerApi`):


https://github.com/mswjs/msw/blob/a851536cc00b23ce7c3a7d71cb5a02ae865adec0/src/setupWorker/setupWorker.ts#L8

Reply:
we can move then `SetupServerWorker` in his glossary file? In this way we have the same structure

Reply:
That's a sound suggestion, @marcosvega91!